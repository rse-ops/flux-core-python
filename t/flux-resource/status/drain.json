{
  "R": {
    "execution": {
      "R_lite": [
        {
          "children": {
            "core": "0-3"
          },
          "rank": "0-3"
        }
      ],
      "expiration": 0,
      "nodelist": [
        "foo[1-4]"
      ],
      "starttime": 0
    },
    "version": 1
  },
  "drain": {
    "0": {
      "reason": "bad node",
      "timestamp": 1607544031.1368911
    },
    "1": {
      "reason": "testing drain",
      "timestamp": 1607544037.2190773
    }
  },
  "exclude": "",
  "offline": "",
  "online": "0-3"
}
