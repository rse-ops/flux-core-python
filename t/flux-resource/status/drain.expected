    STATUS NNODES RANKS           NODELIST
     avail      2 2-3             foo[3-4]
   drained      2 0-1             foo[1-2]
